pkg/config/config.go

```diff
@@ -58,88 +62,6 @@ var (
 	DefaultImage = "gcr.io/heptio-images/sonobuoy:" + buildinfo.Version
 )
 
-///////////////////////////////////////////////////////
-// Note: The described resources are a 1:1 match
-// with kubectl UX for consistent user experience.
-// xref: https://kubernetes.io/docs/api-reference/v1.11/
-///////////////////////////////////////////////////////
-
-// ClusterResources is the list of API resources that are scoped to the entire
-// cluster (ie. not to any particular namespace)
-var ClusterResources = []string{
-	"CertificateSigningRequests",
-	"ClusterRoleBindings",
-	"ClusterRoles",
-	"ComponentStatuses",
-	"CustomResourceDefinitions",
-	"Nodes",
-	"PersistentVolumes",
-	"PodSecurityPolicies",
-	"ServerGroups",
-	"ServerVersion",
-	"StorageClasses",
-}
-
-// NamespacedResources is the list of API resources that are scoped to a
-// kubernetes namespace.
-var NamespacedResources = []string{
-	"ConfigMaps",
-	"ControllerRevisions",
-	"CronJobs",
-	"DaemonSets",
-	"Deployments",
-	"Endpoints",
-	"Events",
-	"HorizontalPodAutoscalers",
-	"Ingresses",
-	"Jobs",
-	"LimitRanges",
-	"NetworkPolicies",
-	"PersistentVolumeClaims",
-	"PodDisruptionBudgets",
-	"PodLogs",
-	"PodTemplates",
-	"Pods",
-	"ReplicaSets",
-	"ReplicationControllers",
-	"ResourceQuotas",
-	"RoleBindings",
-	"Roles",
-	"ServiceAccounts",
-	"Services",
-	"StatefulSets",
-}
-
-// NamespacedDefaults is the default-list of API resources to gather
-// TODO (tstclair): Clean up defaulting, this is temporary
-var NamespacedDefaults = []string{
-	"ConfigMaps",
-	"ControllerRevisions",
-	"CronJobs",
-	"DaemonSets",
-	"Deployments",
-	"Endpoints",
-	//"Events",
-	//"HorizontalPodAutoscalers",
```

Comment:
Why was this skipped at all?

Reply:
Kept for historical sake; not worth a fight at the current time. This stuff will change when we make it its own plugin.